cppcache/integration-test/testSystemProperties.cpp

```diff
@@ -51,7 +51,7 @@ BEGIN_TEST(DEFAULT)
     ASSERT(
         systemProperties->statisticsSampleInterval() == std::chrono::seconds(1),
         "expected 1");
-    ASSERT(systemProperties->statisticsEnabled() == true, "expected true");
+    ASSERT(systemProperties->statisticsEnabled() == false, "expected true");
```

Comment:
The string description should now be "expected false," no?

Reply:
Ha! Details!